### ISSUE TITLE
(🐞) error code that is enabled in the config file can't be disabled on the CLI (with `--disable-error-code`)

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#16628]
I think it would be intuitive for commandline arguments to override config file options, but the way that `enable/disable-error-code` works isn't.

`pyproject.toml`
```py
[tool.mypy]
enable_error_code = "operator"
```
```console
> mypy --disable-error-code operator -c "1 + ''"
test.py:1: error: Unsupported operand types for + ("int" and "str")  [operator]
Found 1 error in 1 file (checked 1 source file)
```
The docs specify how these flags interact with themselves, but don't mention the ones in the config file.
```
--enable-error-code
This flag allows enabling one or multiple error codes globally. See [Error codes](https://mypy.readthedocs.io/en/stable/error_codes.html#error-codes) for more information.

Note: This flag will override disabled error codes from the [--disable-error-code](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-disable-error-code) flag.
```

I think it would be expected that t
